tests/utils/test_dispatcher.py

```diff
@@ -87,3 +91,33 @@ def bar(c, d):
 
     dispatcher = Dispatcher([foo, bar], input_dict={"a": 1, "b": 2, "c": 3, "d": 4, "e": 5})
     assert dispatcher({"a": 1, "b": 2}) == 3
+
+
+def test_call_with_status_db_functions(
```

Comment:
Maybe add a docstring and the GIVEN, WHEN, THEN sections for better readability

Reply:
Done in my last commit